### ISSUE TITLE
Have Inspector show details about Marionette Controllers and Marionette Objects

### ISSUE THREAD

**Issue by mindpivot** [marionettejs/marionette.inspector#225]
First off, just found this thing after using Backbone Inspector to do my Marionette dev for who knows how long and... THANKS!!! Whew, like a breath of fresh air.

I am working on an app which heavily uses Marionette's Controllers and Objects (Objects as route-less controllers essentially, as per Marionette documentation). It would be nice to be able to inspect their properties and lifecycle events like the other objects. I do feel like there's a gap when I'm looking at a bunch of Models, Views, etc but am unable in the same interface to inspect the objects which orchestrate them.

I've managed to work around it to an extent by exposing functionality of those Controllers and Objects via Backbone Radio but it would be nice to see them in context.

Thanks!

**Comment by jasonLaster** [comment]
Yup. I agree. It'd be nice to see an extra architecture tab for some of this stuff.

I'll put together a patch for how this could be done, then hopefully someone can go in and do the heavy lifting. Also happy to pair with someone and get them started

**Comment by mindpivot** [comment]
As I just found this plugin (still kicking myself for taking so long) I doubt I'd be of any use contributing to the code but I will gladly contribute cycles to testing and documentation.
